.eslintrc.js

```diff
@@ -0,0 +1,160 @@
+const error = 2;
+const warn = 1;
+const ignore = 0;
+
+module.exports = {
```

Comment:
@ghengeveld suggested this file is OTT. I'm happy to change it to whatever makes sense, doesn't need to be the same as the monorepo.

Reply:
It's mostly copied from storybook's

Reply:
🤷‍♂